src/functionalTest/resources/1111006_2.metadata.json

```diff
@@ -17,7 +17,7 @@
         "ocr_data": "ew0KICAiTWV0YWRhdGFfZmlsZSI6IFsNCgkgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJyaWdodCB0byBhcHBlYWxfTXJuIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIkZhbHNlIiB9LA0KCSB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImNvbnRhaW5zX01ybiIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICJUcnVlIiB9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJiZW5lZml0VHlwZV9kZXNjcmlwdGlvbiIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiQ2l2aWwgRGl2b3JjZSBDYXNlIn0sDQoJIHsgbWV0YWRhdGFfZmllbGRfbmFtZTogImFwcGVsbGFudF90aXRsZSIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiTXIifSwNCgkgeyBtZXRhZGF0YV9maWVsZF9uYW1lOiAiYXBwZWxsYW50X2ZpcnN0TmFtZSIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiU3VkaGlyIFPMtmjMtnLMtmXMtnPMtnTMtm7MtmHMtiJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBlbGxhbnRfbGFzdE5hbWUiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIlNocmVzdG5hIn0sDQoJIHsgbWV0YWRhdGFfZmllbGRfbmFtZTogImFwcGVsbGFudF9hZGRyZXNzIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICIyNCBPbWVnYSBTdC4gdV9rXyBMb25kb24gVUsifSwNCgkgeyBtZXRhZGF0YV9maWVsZF9uYW1lOiAiYXBwZWxsYW50X3Bvc3Rjb2RlIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICJENDE3QlMifSwNCgkgeyBtZXRhZGF0YV9maWVsZF9uYW1lOiAiYXBwZWxsYW50X3Bob25lIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICIwNzg5NDQ5MTEwNiJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBlbGxhbnRfbW9iaWxlIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICIifSwNCgkgeyBtZXRhZGF0YV9maWVsZF9uYW1lOiAiYXBwZWxsYW50X2RvYiIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiMTEvMTAvMTk4MSJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBlbGxhbnRfbmlGYWxzZSIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiNDdCQVM3NjU0In0sDQoJIHsgbWV0YWRhdGFfZmllbGRfbmFtZTogImlzX2FwcG9pbnRlZSIsIG1ldGFkYXRhX2ZpZWxkX3ZhbHVlOiAiVHJ1ZSJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBvaW50ZWVfdGl0bGUiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIk1yIn0sDQoJIHsgbWV0YWRhdGFfZmllbGRfbmFtZTogImFwcG9pbnRlZV9maXJzdE5hbWUiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIlNvcGhpYSJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBvaW50ZWVfbGFzdE5hbWUiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIlNvbGUifSwNCgkgeyBtZXRhZGF0YV9maWVsZF9uYW1lOiAiYXBwb2ludGVlX2FkZHJlc3MiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIk4vQSJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBvaW50ZWVfcG9zdGNvZGUiLCBtZXRhZGF0YV9maWVsZF92YWx1ZTogIkQxNzFZNyJ9LA0KCSB7IG1ldGFkYXRhX2ZpZWxkX25hbWU6ICJhcHBvaW50ZWVfZG9iIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICIxNC8xMC8xOTgwIn0sDQoJIHsgbWV0YWRhdGFfZmllbGRfbmFtZTogImFwcG9pbnRlZV9uaUZhbHNlIiwgbWV0YWRhdGFfZmllbGRfdmFsdWU6ICI0NDE3NjU0M0IifSwNCgkNCiAgXQ0KICB9",
         "file_name": "1111006.pdf",
         "notes": "",
-        "document_type": "SSC1"
+        "document_type": "Cherished"
```

Comment:
Unfortunately this is going to change now and we are going to get document type(Other, Cherished and SSCS1) and document subtype with same value initially. Later Exela is going to send only Other and Cherished and everything else goes into subtype. This is awaiting confirmation from Exela.

Reply:
Enum can be extended without a sweat. Let's get final confirmation even though task has already been set